Doc/using/mac.rst

```diff
@@ -159,7 +159,7 @@ https://riverbankcomputing.com/software/pyqt/intro.
 Distributing Python Applications on the Mac
 ===========================================
 
-The "Build Applet" tool that is placed in the MacPython 3.6 folder is fine for
```

Comment:
Thanks for catching this.  The Build Applet tool does not exist anymore.  So this whole paragraph should simply be deleted.